x-pack/test/security_solution_cypress/cypress/tasks/create_new_rule.ts

```diff
@@ -557,7 +557,7 @@ export const fillDefineNewTermsRuleAndContinue = (rule: NewTermsRuleCreateProps)
 
 export const fillEsqlQueryBar = (query: string) => {
   // eslint-disable-next-line cypress/no-force
-  cy.get(ESQL_QUERY_BAR_INPUT_AREA).type(query, { force: true });
+  cy.get(ESQL_QUERY_BAR_INPUT_AREA).should('not.be.disabled').type(query, { force: true });
```

Comment:
Any way we can do without the `force:true`. If I recall correctly, it's a pattern we're trying to move away from.

Reply:
Interestingly enough, it was added here: https://github.com/elastic/kibana/pull/171720/commits/31fdcb3661faebaf1f903088d37790116f649a35

Looks like w/o `force:true` it caused flaky tests

Reply:
Tried to remove it, but tests started to fail consistently. Likely, something has changed in ESQL compnent, that now requires this clause

Reply:
Ah ok. Thanks for investigating this one.